### DISCUSSION TITLE
Homarr : Home Server Dashboard

### DISCUSSION THREAD

**Discussion by TommySharpNZ** [MichaIng/DietPi#7208] [Software Request]
Would be cool if we could get Homarr added as a software option at some point. Seems to be a similar options to Homer.
https://homarr.dev/

**Comment by Joulinar** [comment]
Usually we don't add Docker container to our software catalogue.

**Reply by MichaIng** [reply]
Here would be a way to install via Node.js bare metal: https://homarr.dev/docs/getting-started/installation/#not-recommended-from-source

**Reply by Joulinar** [reply]
Yes with a huge yellow `caution - not recommended` box 😜

**Reply by MichaIng** [reply]
Yeah, and it explains the reasons, which are all not a problem when done right, as local module, just like Node-RED. The docs address "modern" end users with the idea they cannot manage anything bug a Docker container. But we can manage proper base metal install for them, and provide one-line update instructions or scripts.

I mean tbh, with Node-RED, there are sometimes issues with conflicts between the system Node and the local module. I don't remember the exact problem(s), but one time it was due to a mix of our Node.js install option ans APT packages, providing additional Node modules on the system level, but for a different Node/npm version, or something like that. But IMO worth to give it a try like that, when implementing it.
